### PR TITLE
test: add test for asset dashboard

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [x] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [ ] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA

### CODE REVIEW

File: apps/web/tests/dashboards/create-asset-dashboard/create-compliance-overview-dashboard.spec.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            await expect(locatorName).toContainText(dashboardName);
+
+            // Todo: to be added
+            // await expect(locatorName).toContainText('public');
+        });
+
+        await test.step('6. Delete dashboard', async () => {
+            const deleteButton = await page.locator('.dashboard-title-icon-buttons-wrapper .delete-button');
+            await deleteButton.click();
+            await page.getByRole('button', { name: 'Confirm' }).click();
```
Comment (sulmoJ) [inline apps/web/tests/dashboards/create-asset-dashboard/create-compliance-overview-dashboard.spec.ts:50]: Question: There is no expect, how can I check the deletion?

Comment (yuda110) [reply apps/web/tests/dashboards/create-asset-dashboard/create-compliance-overview-dashboard.spec.ts:50]: Oh, I'll add expect. Thanks!!!

File: apps/web/tests/dashboards/create-dashboards/entire-workspace-dashboard.spec.ts [hunk 2]
```diff
@@ -34,14 +34,20 @@ test.describe('Entire Workspace Dashboard', () => {
...
 
             // Todo: to be added
             // await expect(locatorName).toContainText('public');
         });
+
+        await test.step('6. Delete dashboard', async () => {
+            const deleteButton = await page.locator('.dashboard-title-icon-buttons-wrapper .delete-button');
+            await deleteButton.click();
+            await page.getByRole('button', { name: 'Confirm' }).click();
+        });
```
Comment (skdud4659) [inline apps/web/tests/dashboards/create-dashboards/entire-workspace-dashboard.spec.ts:51]: Suggestion: It seems better to validate each test in a single test step using 'expect'!

File: apps/web/tests/dashboards/create-dashboards/single-project-dashboard.spec.ts [hunk 3]
```diff
@@ -46,5 +46,11 @@ test.describe('Single Project Dashboard', () => {
             // Todo: to be added
             // await expect(locatorName).toContainText('public');
         });
+
+        await test.step('6. Delete dashboard', async () => {
+            const deleteButton = await page.locator('.dashboard-title-icon-buttons-wrapper .delete-button');
+            await deleteButton.click();
+            await page.getByRole('button', { name: 'Confirm' }).click();
+        });
```
Comment (skdud4659) [inline apps/web/tests/dashboards/create-dashboards/single-project-dashboard.spec.ts:54]: Suggestion: same suggestion!

File: apps/web/tests/dashboards/create-dashboards/entire-workspace-dashboard.spec.ts (hunk 2 above)
Comment (yuda110) [reply apps/web/tests/dashboards/create-dashboards/entire-workspace-dashboard.spec.ts:51]: Thank you for suggestion! I'll update that!

**Pull request closed without merging**